core/src/SYCL/Kokkos_SYCL_ParallelReduce_Range.hpp

```diff
@@ -322,9 +322,23 @@ class Kokkos::Impl::ParallelReduce<
         auto reduction_lambda = reduction_lambda_factory(
             local_mem, num_teams_done, results_ptr, values_per_thread);
 
+#if defined(__INTEL_LLVM_COMPILER) && __INTEL_LLVM_COMPILER >= 20230100
```

Comment:
Why are the guards different on this one?
All other occurrences so far were doing `#ifdef SYCL_EXT_ONEAPI_KERNEL_PROPERTIES`

Reply:
Fixed. Most have been an artifact from rebasing/merging.